### DISCUSSION TITLE
Where can I find cool README templates?

### DISCUSSION THREAD

**Discussion by gabrielbaltar** [community/community#123843] [Repositories] [answered]
### Select Topic Area

Question

### Body

I need to create a README for my personal project and I'd like to use a pre-made template to save time...

**Comment by gitstua** [comment, marked as answer]
This is an interesting list to look through.. https://github.com/matiassingers/awesome-readme

Also, @mishmanners has some great ideas in this space... https://dev.to/mishmanners/github-readme-for-easy-to-understand-code-2b5k

**Comment by saifulislam07** [comment]
i created my README file but its not showing as like i want... :(

**Comment by subhadeeproy3902** [comment]
To create a README using a pre-made template, you can follow these steps:

1. **Visit the GitHub README template repository**: There are many repositories on GitHub that provide README templates. A popular one is [this collection of README templates](https://github.com/othneildrew/Best-README-Template).

2. **Fork or download the template**: Fork the repository to your own GitHub account or download the template files to your local machine.

3. **Customize the template**: Edit the template to include information specific to your project. Most templates have placeholders for project name, description, installation instructions, usage examples, and more.

4. **Add the README to your project**: Once you've customized the README template, add it to the root of your project directory and commit the changes.

Alternatively, you can use GitHub's built-in README generator:

1. **Navigate to your repository** on GitHub.
2. Click on **"Add a README"** if you don’t have one yet, or **edit the existing README** file.
3. GitHub provides a basic template that you can fill out with your project details.

I hope this helps you get started quickly with your README!


Accept the answer if u liked it :)

**Comment by Alexix505** [comment]
you can get it through

https://gh-stats-gen.vercel.app/
